src/services/services.ts

```diff
@@ -31,6 +31,8 @@ namespace ts {
     /** The version of the language service API */
     export const servicesVersion = "0.5";
 
+    const ruleProvider: formatting.RulesProvider = new formatting.RulesProvider();

```

Comment:
We could call this in server/editorServices.ts to initialize the provider instead of doing it here.